Deployment.md

```diff
@@ -36,7 +36,7 @@ Open a bash shell and execute [inception.sh](deployment/inception.sh)
 
 ```bash
 cd /deployment
-chmod 775 *
```

Comment:
shouldn't this be `555`?

Reply:
You don't want people to be able to write? 7=wrx 5 = r-x